pkgs/by-name/ed/eduli/package.nix

```
@@ -22,13 +22,9 @@ stdenvNoCC.mkDerivation rec {
   '';
 
   meta = {
-    description = "The MOE Li Font, a clerical Chinese font by the Ministry of Education, ROC (Taiwan)";
+    description = "The MOE Li font, a clerical-script-style Chinese character typeface";
```

Current version of `pkg/README.md` forbids `meta.description` starting with an article.

```suggestion
    description = "MOE Li font, a clerical-script-style Chinese character typeface";
```